<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PearRay Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="ViewWidget" name="viewWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuDockWidgets">
     <property name="title">
      <string>Dock widgets</string>
     </property>
     <addaction name="actionShowDockWidgets"/>
     <addaction name="actionHideDockWidgets"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>Toolbars</string>
     </property>
     <addaction name="actionShowToolbars"/>
     <addaction name="actionHideToolbars"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionSelection_Tool"/>
    <addaction name="actionPan_Tool"/>
    <addaction name="actionZoom_Tool"/>
    <addaction name="actionCrop_Tool"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Zoom"/>
    <addaction name="actionFit_Image_into_Area"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="menuToolbars"/>
    <addaction name="menuDockWidgets"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenScene"/>
    <addaction name="separator"/>
    <addaction name="actionExportImage"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuExternal_Links">
     <property name="title">
      <string>External Links</string>
     </property>
     <addaction name="actionWebsite"/>
    </widget>
    <addaction name="actionShortcuts"/>
    <addaction name="actionQuickGuide"/>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="menuExternal_Links"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionStartRender"/>
    <addaction name="actionRestartRender"/>
    <addaction name="actionCancelRender"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="systemPropertyDockWidget">
   <property name="windowTitle">
    <string>System Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="content">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="SystemPropertyView" name="systemPropertyView" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="spectrumDockWidget">
   <property name="windowTitle">
    <string>Spectrum</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>778</width>
          <height>69</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="SpectrumWidget" name="spectrumWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outlineDockWidget">
   <property name="windowTitle">
    <string>Outline</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="outlineView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="entityDetailsDockWidget">
   <property name="windowTitle">
    <string>Entity Details</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="EntityDetailsView" name="entityDetailsView" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenScene"/>
   <addaction name="actionExportImage"/>
   <addaction name="separator"/>
   <addaction name="actionStartRender"/>
   <addaction name="actionRestartRender"/>
   <addaction name="actionCancelRender"/>
   <addaction name="separator"/>
   <addaction name="actionSelection_Tool"/>
   <addaction name="actionPan_Tool"/>
   <addaction name="actionZoom_Tool"/>
   <addaction name="actionCrop_Tool"/>
   <addaction name="separator"/>
   <addaction name="actionReset_Zoom"/>
   <addaction name="actionFit_Image_into_Area"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionQuickGuide">
   <property name="text">
    <string>Quick Guide</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_home_black_24dp_2x.png</normaloff>:/ic_home_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Website</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionHideDockWidgets">
   <property name="text">
    <string>Hide all</string>
   </property>
  </action>
  <action name="actionShowDockWidgets">
   <property name="text">
    <string>Show all</string>
   </property>
  </action>
  <action name="actionShowToolbars">
   <property name="text">
    <string>Show all</string>
   </property>
  </action>
  <action name="actionHideToolbars">
   <property name="text">
    <string>Hide all</string>
   </property>
  </action>
  <action name="actionExportImage">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_file_download_black_24dp_2x.png</normaloff>:/ic_file_download_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Export Image</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionOpenScene">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_nature_people_black_24dp_2x.png</normaloff>:/ic_nature_people_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Open scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStartRender">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_play_arrow_black_24dp_2x.png</normaloff>:/ic_play_arrow_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Start Render</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCancelRender">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_stop_black_24dp_2x.png</normaloff>:/ic_stop_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Cancel Render</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_aspect_ratio_black_24dp_2x.png</normaloff>:/ic_aspect_ratio_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom/Pan</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionFit_Image_into_Area">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_settings_overscan_black_24dp_2x.png</normaloff>:/ic_settings_overscan_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Fit Image into Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_zoom_in_black_24dp_2x.png</normaloff>:/ic_zoom_in_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_zoom_out_black_24dp_2x.png</normaloff>:/ic_zoom_out_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionPan_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_pan_tool_black_24dp_2x.png</normaloff>:/ic_pan_tool_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Pan Tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSelection_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_colorize_black_24dp_2x.png</normaloff>:/ic_colorize_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Selection Tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionZoom_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_search_black_24dp_2x.png</normaloff>:/ic_search_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionCrop_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_crop_black_24dp_2x.png</normaloff>:/ic_crop_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Crop Tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionRestartRender">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_replay_black_24dp_2x.png</normaloff>:/ic_replay_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Restart Render</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewWidget</class>
   <extends>QWidget</extends>
   <header>widgets/ViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectrumWidget</class>
   <extends>QWidget</extends>
   <header>widgets/SpectrumWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EntityDetailsView</class>
   <extends>QWidget</extends>
   <header>widgets/EntityDetailsView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SystemPropertyView</class>
   <extends>QWidget</extends>
   <header>widgets/SystemPropertyView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
